addons/website_twitter/models/twitter.py

```diff
@@ -41,52 +40,51 @@ def _request(self, website, url, params=None):
+                                          ('twitter_screen_name', '!=', False)])
+        _logger.debug("Refreshing tweets for website IDs: %r", ids.id)
+        website.fetch_favorite_tweets()
 
-    def fetch_favorite_tweets(self, cr, uid, ids, context=None):
-        website_tweets = self.pool['website.twitter.tweet']
+    @api.multi
+    def fetch_favorite_tweets(self):
+        website_tweets = self.env['website.twitter.tweet']
         tweet_ids = []
-        for website in self.browse(cr, uid, ids, context=context):
+        for website in self.search([]):
             if not all((website.twitter_api_key, website.twitter_api_secret,
                        website.twitter_screen_name)):
                 _logger.debug("Skip fetching favorite tweets for unconfigured website %s",
                               website) 
                 continue
             params = {'screen_name': website.twitter_screen_name}
-            last_tweet = website_tweets.search_read(
-                    cr, uid, [('website_id', '=', website.id),
-                              ('screen_name', '=', website.twitter_screen_name)],
-                    ['tweet_id'],
-                    limit=1, order='tweet_id desc', context=context)
+            last_tweet = website_tweets.search_read([('website_id', '=', website.id),
+                                                     ('screen_name', '=', website.twitter_screen_name)],
+                                                     ['tweet_id'],
+                                                     limit=1, order='tweet_id desc')
             if last_tweet:
                 params['since_id'] = int(last_tweet[0]['tweet_id'])
             _logger.debug("Fetching favorite tweets using params %r", params)
             response = self._request(website, REQUEST_FAVORITE_LIST_URL, params=params)
             for tweet_dict in response:
                 tweet_id = tweet_dict['id'] # unsigned 64-bit snowflake ID
-                tweet_ids = website_tweets.search(cr, uid, [('tweet_id', '=', tweet_id)])
+                tweet_ids = website_tweets.search([('tweet_id', '=', tweet_id)]).ids
                 if not tweet_ids:
                     new_tweet = website_tweets.create(
-                            cr, uid,
                             {
                               'website_id': website.id,
                               'tweet': json.dumps(tweet_dict),
                               'tweet_id': tweet_id, # stored in NUMERIC PG field 
                               'screen_name': website.twitter_screen_name,
-                            },
-                            context=context)
+                            })
                     _logger.debug("Found new favorite: %r, %r", tweet_id, tweet_dict)
-                    tweet_ids.append(new_tweet)
+                    tweet_ids.append(new_tweet.id)
         return tweet_ids
 
+    @api.multi
```

Comment:
1. why the decorator?
2. why would it be `multi` since `self` isn't even used in the method?